overlord/devicestate/devicestate.go

```diff
@@ -1479,3 +1479,10 @@ func InstallSetupStorageEncryption(st *state.State, label string, onVolumes map[
 
 	return chg, nil
 }
+
+func Reseal(st *state.State) *state.Change {
+	chg := st.NewChange("reseal-device", i18n.G("Reseal the device"))
```

Comment:
The change may need to be accounted for in conflicts. I suppose for simplicity we may disallow any other changes from running, cc @pedronis

Reply:
Out of curiosity, what conflict are you thinking of?

Reply:
Wondering if there could be a potential issue if the there is a create recovery system create or remodel or gadget update. IIRC the remodel and recovery wee made exclusive in https://github.com/snapcore/snapd/blob/master/overlord/snapstate/conflict.go#L147 not sure if that has changed.